src/main/java/com/files/AddressValidator.java

```diff
@@ -0,0 +1,47 @@
+package com.files;
+
+import com.log.Logs;
+import com.setup.Setup;
+
+import java.net.URL;
+import java.util.logging.Logger;
+
+public class AddressValidator {
+
+    public UrlValidator validate(URL url) {
+
+        Setup setup = new Setup();
+
+        Logger log = Logger.getLogger(Logs.class.getName());
+        log.info("Add new url in progress for validation: " + url);
+
+        UrlValidator urlValidator = new UrlValidator();
+
+        char[] ex = url.toString().toCharArray();
```

Comment:
@PavelNejinskiy please move this logic to `Image` interface as default method name `extension`